packages/cli/src/controllers/debug.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Service } from 'typedi';
+import { Get, RestController } from '@/decorators';
+import { ActiveWorkflowRunner } from '@/ActiveWorkflowRunner';
+import { MultiMainSetup } from '@/services/orchestration/main/MultiMainSetup.ee';
+import { WorkflowRepository } from '@/databases/repositories/workflow.repository';
+import { In } from 'typeorm';
+
+@RestController('/debug')
+@Service()
+export class DebugController {
+	constructor(
+		private readonly multiMainSetup: MultiMainSetup,
+		private readonly activeWorkflowRunner: ActiveWorkflowRunner,
+		private readonly workflowRepository: WorkflowRepository,
+	) {}
+
+	@Get('/multi-main-setup')
+	async getMultiMainSetupDetails() {
+		const activeWorkflows = await this.workflowRepository.find({
+			select: ['id', 'name'],
+			where: { id: In(this.activeWorkflowRunner.allActiveInMemory()) },
+		});
+
+		const activationErrors = await this.activeWorkflowRunner.getAllWorkflowActivationErrors();
```

Comment:
Can we also add the current value of the redis key?

Reply:
Good idea

Reply:
I mean not just the key but also the value wdyt?